cmd/istioctl/main.go

```diff
@@ -319,7 +319,7 @@ func init() {
 	rootCmd.PersistentFlags().StringVarP(&kubeconfig, "kubeconfig", "c", defaultKubeconfig,
 		"Kubernetes configuration file")
 
-	rootCmd.PersistentFlags().StringVarP(&istioNamespace, "istioNamespace", "ns", kube.IstioNamespace,
+	rootCmd.PersistentFlags().StringVarP(&istioNamespace, "istioNamespace", "i", kube.IstioNamespace,
```

Comment:
thanks, though we may want to keep -i for something more likely to be changed by our users (like for some -include of some sort... afaik only test scripts would ever change the istionamespace ?)
you don't like `-ins` ?